openmdao/core/system.py

```diff
@@ -2605,15 +2618,6 @@ def _solve_linear(self, vec_names, mode):
             list of names of the right-hand-side vectors.
         mode : str
             'fwd' or 'rev'.
-
```

Comment:
Well I mean you deleted the "Returns" part of the docstring, but i am arguing that it should be restored because the child overrides return that stuff.